math/UsefulMathFunctions.cpp

```diff
@@ -0,0 +1,188 @@
+
+#include <bits/stdc++.h>
```

Comment:
Please remove this header because it is quite linux specific and increases the compile time.